feature-libs/cart/quick-order/components/cart-quick-order-form/cart-quick-order-form.component.html

```diff
@@ -153,7 +163,15 @@
           ></cx-form-errors>
 
           <cx-form-errors
-            *cxFeature="'formErrorsDescriptiveMessages'"
+            *cxFeature="'formErrorsDescriptiveMessages && a11yQTY2Quantity'"
```

Comment:
Does this work? I don't think `cxFeature` has been designed to handle conditional logic.

Reply:
I've just updated it.